### PR TITLE
multi extension support for config

### PR DESCRIPTION
**Pull request by mpeck**
This adds support for the `.yml`, `.yaml` and `.json` extensions.

reference https://github.com/operable/cog/issues/413

### CODE REVIEW

File: lib/relay/bundle/installer.ex [hunk 1]
```diff
@@ -253,7 +255,9 @@ defmodule Relay.Bundle.Installer do
 
   defp expand_bundle(bf, config) when is_binary(bf) do
     bundle_root = Application.get_env(:relay, :bundle_root)
-    install_path = build_install_dest(bundle_root, config, true)
+    bundle_path = String.replace_trailing(bf, ".locked", "")
+    [bundle_ext] = Regex.run(~r(\.[a-zA-Z]*$), bundle_path)
```
Comment (vanstee) [inline lib/relay/bundle/installer.ex]: Any reason not to use `Path.extname` here?

File: lib/relay/bundle/installer.ex [hunk 2]
```diff
@@ -253,7 +255,9 @@ defmodule Relay.Bundle.Installer do
 
   defp expand_bundle(bf, config) when is_binary(bf) do
     bundle_root = Application.get_env(:relay, :bundle_root)
-    install_path = build_install_dest(bundle_root, config, true)
+    bundle_path = String.replace_trailing(bf, ".locked", "")
```
Comment (vanstee) [inline lib/relay/bundle/installer.ex]: Might be more intention revealing to use `Path.basename(bf, "locked")` here.

File: lib/relay/bundle/scanner.ex [hunk 3]
```diff
@@ -125,8 +125,12 @@ defmodule Relay.Bundle.Scanner do
 
   defp pending_bundle_files() do
     pending_path = Application.get_env(:relay, :pending_bundle_root)
+    # Generates a string like '{"*.yaml", "*.yml", "*.json"}' which is used with
+    # Path.wildcard/2 to search for multiple file types
+    skinny_bundles = "{#{Enum.map_join(Spanner.skinny_bundle_extensions, ",", &("*#{&1}"))}}"
```
Comment (vanstee) [inline lib/relay/bundle/scanner.ex]: I think `*.{yml,yaml,json}` might be a little easier to generate here.

File: lib/relay/bundle/installer.ex (hunk 1 above)
Comment (mpeck) [inline lib/relay/bundle/installer.ex]: Why yes there is. And that reason is that I did not know about it :)

File: lib/relay/bundle/installer.ex (hunk 2 above)
Comment (mpeck) [inline lib/relay/bundle/installer.ex]: Another thing I learned today :)

**Pull request merged**